### DISCUSSION TITLE
How to present a  mysql `COMMENT`?

### DISCUSSION THREAD

**Discussion by tsui66** [prisma/prisma#7278] [Q&A] [answered]
How to /// comments are present of mysql schema `COMMENT` as below
![image](https://user-images.githubusercontent.com/2019808/119627366-b3e04000-be3e-11eb-8107-89d320017ae6.png)

**Comment by ryands17** [comment, marked as answer]
@Quinton 👋 

Currently comments are not supported directly in `schema.prisma` but you should be able to add them to the migration `.sql` created.

**Reply by janpio** [reply]
Has been requested before as a feature at https://github.com/prisma/prisma/issues/4818
